Create nl.json

Voxxie opened the pull request:
Added the dutch translation

SukramJ commented:
Thanks for creating a translation.
Please be aware that this integration is in a very early stage, and that some keys might change that affects translations.

Due to your latest merge your branch contains to many commits that are not related to your translation.
Please rebase your branch to get rid of these additional commits.

Voxxie commented:
Sorry github is quite new for me. 
I updated the file and rebased my request.

I know that its still in an early stage, i am learning how custom_components for HA work and am trying to help where I can. 
So by following your adjustments to this component i am learning somethings ;)
